src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -806,4 +811,113 @@ private function addVariantWithPriceToOrder(OrderInterface $order, ProductVarian
 
         $order->addItem($item);
     }
+
+    /**
+     * @param int $numberOfCustomers
+     * @param int $numberOfOrders
+     * @param string $total
+     * @param bool $isFulfilled
+     */
+    private function createOrders(
+        int $numberOfCustomers,
+        int $numberOfOrders,
+        string $total,
+        bool $isFulfilled = false
+    ): void {
+        $customers = $this->generateCustomers($numberOfCustomers);
+        $sampleProductVariant = $this->sharedStorage->get('variant');
+        $total = $this->getPriceFromString($total);
+
+        for ($i = 0; $i < $numberOfOrders; $i++) {
```

Comment:
We could think about using ExampleFacotories to create test orders but that an idea for later